helpers/index.js

```diff
@@ -54,3 +54,9 @@ export const timezoneOffsetLocal = (timezone) => {
 
   return localOffset - offset;
 };
+
+export const arrayRange = (start, stop, step) =>
```

Comment:
generates an array of numbers, incremented by the step value, between a range. i.e.

start: 1
stop: 5
step: 1

would produce `[1,2,3,4,5]`